cisco_umbrella_dns/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - cisco_umbrella_dns
```

Comment:
```suggestion
# CHANGELOG - cisco_umbrella_dns

<!-- towncrier release notes start -->
```

Please add this. We will fix our template for new integrations soon, but until then we have to keep adding this manually. Apologies for the inconvenience.

Reply:
Done